src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts

```diff
@@ -0,0 +1,136 @@
+
+const helperSteps: Helper = new Helper()
+const userSteps: UserSteps = new UserSteps()
+const claimantResponseSteps: ClaimantResponseSteps = new ClaimantResponseSteps()
+const checkAndSendPage: ClaimantCheckAndSendPage = new ClaimantCheckAndSendPage()
+const confirmationPage: ClaimantConfirmation = new ClaimantConfirmation()
+
+Feature('Claimant Response').retry(3)
+
+Scenario('I can as a claimant view the defendants full admission with immediate payment @citizen @admissions', async (I: I) => {
+  const testData = await EndToEndTestData.prepareData(I, PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)
+  testData.paymentOption = PaymentOption.IMMEDIATELY
+  // as defendant
+  helperSteps.finishResponseWithFullAdmission(testData)
+  I.click('Sign out')
+  // as claimant
+  userSteps.login(testData.claimantEmail)
+  claimantResponseSteps.viewClaimFromDashboard(testData.claimRef, false)
+  I.see(testData.claimRef)
+  I.see('The defendant said they’ll pay you immediately')
+  I.click('My account')
+  I.see(testData.claimRef)
+  I.see('The defendant admits they owe all the money. They’ve said that they will pay immediately.')
+})
+
+Scenario('I can as a claimant accept the defendants full admission by set date with settlement agreement and accepting defendants payment method @citizen @admissions', async (I: I) => {
+  const testData = await EndToEndTestData.prepareData(I, PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)
+  testData.paymentOption = PaymentOption.BY_SET_DATE
+  // as defendant
+  helperSteps.finishResponseWithFullAdmission(testData)
+  I.click('Sign out')
+  // as claimant
+  userSteps.login(testData.claimantEmail)
+  claimantResponseSteps.acceptSettlementFromDashboardWhenAcceptPaymentMethod(testData)
+  checkAndSendPage.verifyFactsForSettlement()
+  checkAndSendPage.checkFactsTrueAndSubmit()
+  I.see('You’ve accepted the repayment plan')
+  confirmationPage.clickGoToYourAccount()
+  I.see(testData.claimRef)
+  I.see('You’ve signed a settlement agreement')
+})
+
+Scenario('I can as a claimant accept the defendants full admission by set date with settlement agreement and rejecting defendants payment method in favour of immediate payment @citizen @admissions @error', async (I: I) => {
+  const testData = await EndToEndTestData.prepareData(I, PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)
+  testData.paymentOption = PaymentOption.BY_SET_DATE
+  testData.claimantPaymentOption = PaymentOption.IMMEDIATELY
+  const claimantResponseTestData = new ClaimantResponseTestData()
+  // as defendant
+  helperSteps.finishResponseWithFullAdmission(testData)
+  I.click('Sign out')
+  // as claimant
+  userSteps.login(testData.claimantEmail)
+  claimantResponseSteps.acceptSettlementFromDashboardWhenRejectPaymentMethod(testData, claimantResponseTestData)
+  checkAndSendPage.verifyFactsForSettlement()
+  checkAndSendPage.checkFactsTrueAndSubmit()
+  I.see('422 - "courtDetermination.courtPaymentIntention : must not be null"')
+  // confirmationPage.clickGoToYourAccount()
```

Comment:
please remove if not needed any more

Reply:
hopefully when fixed these can just be commented out for test to pass - i can get rid of though? as will need looking at anyway

Reply:
ah ok. this is one of the flows that error! understood